wpeverest/everest-forms
# Enhancement - Enhanced select options.

### All Submissions:






### Changes proposed in this Pull Request:

Enhanced select options 'Enable Search' and 'Enable Checkbox' are added..

### How to test the changes in this Pull Request:

1. Verify, whether all enhanced select options enable select all, enable search and enable checkbox works properly or not.


### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Enhanced select options.
